e2e/tests/init_test.go

```diff
@@ -61,7 +61,10 @@ var (
 	flagFeaturesPath string
 	keyRingFlag      = "--keyring-backend=test"
 	chainIdFlag      = "--chain-id=poktroll"
-	appGateServerUrl = "http://localhost:42069" // Keeping localhost by default because that is how we run the tests on our machines locally
+	// Keeping localhost by default because that is how we run the tests on our machines locally
+	// appGateServerUrl is pointing to a non-sovereign app gate server so multiple
+	// apps could relay through it.
+	appGateServerUrl = "http://localhost:42079"
```

Comment:
TL;DR; we're using the gateway instead of the sovereign app.

Since apps can no longer stake for more that 1 service. In order to be able to stake to 2 services (`anvil` and `rest`) in the suite, 2 apps -> 2 appgateservers are now needed. We switch to the gateway to not force scaling gateways.